app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +190,40 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    let themisPublicationActivities = await this.args.meeting.themisPublicationActivities;
+    let themisPublicationActivity = themisPublicationActivities.find((activity) => activity.scope.includes('documents'));
```

Comment:
Maybe better to use `this.store.query` to ensure we're not looking at stale data?

Reply:
Use a value imported from constants instead of hardcoded `documents` value (already used in other places in this file)